libsmtutil/Z3Interface.h

```diff
@@ -49,7 +49,7 @@ class Z3Interface: public SolverInterface, public boost::noncopyable
 
 	// Z3 "basic resources" limit.
 	// This is used to make the runs more deterministic and platform/machine independent.
-	static int const resourceLimit = 12500000;
+	static int const resourceLimit = 1000000;
```

Comment:
Can this be made configurable, so that for tests we use a certain setting, but higher for users for usability?

Reply:
Yea, I was thinking the same, just that the setting should be `timeout` instead, since `rlimit` is just hard to control.

Reply:
@axic tracked here https://github.com/ethereum/solidity/issues/9876